inst/NEWS.Rd

```diff
@@ -3,6 +3,13 @@
 \newcommand{\ghpr}{\href{https://github.com/eddelbuettel/nanotime/pull/#1}{##1}}
 \newcommand{\ghit}{\href{https://github.com/eddelbuettel/nanotime/issues/#1}{##1}}
 
+\section{Changes in version 0.2.5 (2019-xx-yy)}{
+  \itemize{
+    \item Add new comparison operators for \code{nanotime} and
+    \code{charcters} (Dirk in \ghpr{54} fixing \ghit{52}).
```

Comment:
Of no importance whatsoever, just in case you personally care, `character` rather than `charcters`...

Reply:
Oops.  But will fix in the next PR -- will quickly convert the tests to tinytest.